Jenkinsfile

```diff
@@ -131,7 +131,9 @@ pipeline {
       }
       steps {
         script {
-          maven cmd: "deploy -Dmaven.test.skip=true -Divy.engine.version='[10.0.0,]'"
+          maven cmd: "deploy -Dmaven.test.skip=true " +
+                  "-Divy.engine.version='[10.0.0,]'" +
+                  "-DdeployToUrl='https://100.demo.ivyteam.io:8443/'"
```

Comment:
Will be 10.demo.ivyteam.io/

Reply:
@ivy-cst  Are you sure? 🤔 
![image](https://user-images.githubusercontent.com/45038833/197135455-959e5a57-d292-4822-a3b9-56928e1ab179.png)

Reply:
Why do you need to wait for 100.demo.ivyteam.io? the current nightly should also be compatible with a release 10 project or not?